### PR TITLE
Added accelerating phi 2 blog.

### PR DESCRIPTION
**Pull request by MaanavD**
Changes staged here: 
https://maanavd.github.io/onnxruntime/blogs

### CODE REVIEW

**Review by natke — COMMENTED** [review]
Looking good! The image sizes are a bit inconsistent - can we unify those?

**Review by sophies927 — DISMISSED** [review]
Made quite a few changes to syntax that I would recommend including before publication!

File: src/routes/blogs/accelerating-phi-2/+page.svx [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+- [Gemma](#gemma)
+- [Conclusion](#conclusion)
+
+<div id="phi-2"/>
+
+# Phi-2
+
+[Phi-2](https://huggingface.co/microsoft/phi-2) is a 2.7 billion parameter transformer model developed by Microsoft. It is an SLM that exhibits excellent reasoning and language comprehension skills. With its small size, Phi-2 is a great platform for researchers, who can explore various aspects such as mechanistic interpretability, safety improvements, and fine-tuning experiments on different tasks.
+
+ONNX Runtime 1.17 introduces kernels changes that support the Phi-2 model, including optimizations for Attention, Multi-Head Attention, Grouped-Query Attention, and RotaryEmbeddingPhi-2. Specifically, support has been added for the following:
```
[the remark this answers is not in the document]
Comment (wangyems) [reply src/routes/blogs/accelerating-phi-2/+page.svx:48]: I think it should be RotaryEmbedding for Phi-2

**Review by parinitarahi — APPROVED** [review]
LGTM

**Review by prasanthpul — APPROVED** [review]

**Pull request merged**